docs/backend-system/building-backends/08-migrating.md

```diff
@@ -1042,3 +1042,44 @@ backend.add(import('@backstage/plugin-auth-backend'));
 backend.add(authModuleGoogleProvider);
 /* highlight-add-end */
 ```
+
+#### Using Legacy Providers
+
+Not all authentication providers have been refactored to support the new backend system.
+
+> You can track the progress of the module migration efforts [here](https://github.com/backstage/backstage/issues/19476).
+> If your authentication provider module is not available yet, you will need to import your backend auth plugin using the legacy helper.
+
+```ts title="packages/backend/src/index.ts"
+import { createBackend } from '@backstage/backend-defaults';
+/* highlight-add-next-line */
+import { makeLegacyPlugin } from '@backstage/backend-common';
+import { coreServices } from '@backstage/backend-plugin-api';
+
+/* highlight-add-start */
+const legacyPlugin = makeLegacyPlugin({
```

Comment:
Ah, you wouldn't actually need to create a custom `legacyPlugin` helper for this. Its purpose is only to provide and map dependencies, the `providerFactories` should be part of the plugin-specific configuration in `./plugins/auth.ts`